### PR TITLE
Refs [TASK][38389] Create UI HomeScreen

### PR DESCRIPTION
**Pull request by vinhmone**
## Related Tickets
- [#Task 38389: Create UI HomeScreen](https://edu-redmine.sun-asterisk.vn/issues/38389)
## WHAT
- 
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/drawable/bg_home_view.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <corners android:radius="20dp" />
+    <stroke
+        android:width="0.2dp"
+        android:color="@color/color_boulder" />
+    <solid android:color="@color/color_frangipani" />
```
Comment (VietKFC) [inline app/src/main/res/drawable/bg_home_view.xml]: ông đặt tên có cả color và radius , ví dụ bg_radius20_boulder.xml để dễ gọi lại

File: app/src/main/res/layout/fragment_home.xml [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+    android:layout_height="match_parent"
+    tools:context=".ui.home.HomeFragment">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <ImageView
+            android:layout_width="0dp"
+            android:layout_height="@dimen/dp_200"
```
Comment (datdt-1935) [inline app/src/main/res/layout/fragment_home.xml]: không fix cứng width-height của view

File: app/src/main/res/layout/fragment_home.xml [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+            android:contentDescription="@null"
+            android:src="@drawable/ic_cancel"
+            app:layout_constraintBottom_toBottomOf="@id/textSearchDateTo"
+            app:layout_constraintEnd_toEndOf="@id/viewItemDate"
+            app:layout_constraintStart_toEndOf="@id/textSearchDateTo"
+            app:layout_constraintTop_toTopOf="@id/textSearchDateTo" />
+
+        <ImageView
+            android:layout_width="0dp"
+            android:layout_height="0.5dp"
```
Comment (datdt-1935) [inline app/src/main/res/layout/fragment_home.xml]: hard code

File: app/src/main/res/layout/fragment_home.xml [hunk 4]
```diff
@@ -0,0 +1,361 @@
...
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ui.home.HomeFragment">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <ImageView
```
Comment (thonc-2135) [inline app/src/main/res/layout/fragment_home.xml:13]: thiếu contentdecription nè

File: app/src/main/res/layout/fragment_home.xml [hunk 5]
```diff
@@ -0,0 +1,361 @@
...
+            app:layout_constraintStart_toStartOf="@id/textDateFrom"
+            app:layout_constraintTop_toBottomOf="@id/textDateFrom"
+            app:layout_constraintWidth_percent="0.08" />
+
+        <TextView
+            android:id="@+id/textSearchDateFrom"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:layout_marginStart="@dimen/dp_10"
+            android:layout_marginEnd="@dimen/dp_10"
```
Comment (thonc-2135) [inline app/src/main/res/layout/fragment_home.xml]:
```suggestion
            android:layout_marginHorizontal="@dimen/dp_10"
```

**Review by datdt-1935 — APPROVED** [review]

**Review by thonc-2135 — APPROVED** [review]

**Review by datdt-1935 — APPROVED** [review]

**Pull request merged**